TESTING.md

```diff
@@ -67,10 +50,11 @@ brew install chromedriver
 
 If your version of Chrome is 61 or later, check `chromedriver --version` to make sure you're up to at least 2.32. If not, use `brew upgrade chromedriver` to update.
  
-Also run the config render script. If you are planning on running the firecloud ui locally, add the local_ui param (it will set the baseUrl to "http://local.broadinstitute.org/". This will render the necessary `application.conf` and `firecloud-account.pem` for the tests. From the `automation` directory:
+Also run the config render script. Configs are common across all test suites and live in [firecloud-automated-testing](https://github.com/broadinstitute/firecloud-automated-testing).  `render-local-env.sh` will clone a branch of custom configs or the default master branch.  
+If you are planning on running the firecloud ui locally, add the local_ui param (it will set the baseUrl to "http://local.broadinstitute.org/". This will render the necessary `application.conf` and `firecloud-account.pem` for the tests. From the `automation` directory:
 
 ```bash
-./render-local-env.sh [working dir] [vault token] [dev] [local_ui]
+./render-local-env.sh [working dir] [vault token] [branch of firecloud-automated-testing] [env] [local_ui]
```

Comment:
Doesn't NEED to happen but do we want to take this opportunity to change the order of these params so the ones that have "defaults" are last, so we don't have to enter them every time regardless?

Reply:
would be possible to provide examples on `render-loca-env.sh` usage? for example, what are params to use if someone wants to run Selenium UI tests on localhost Chrome, in IntelliJ, against the dev branch, on Fiab? Or, if you changed UI Clojure code, you want to test changes on local Chrome and on Fiab? At the beginning I was confused about meaning of param [local_ui]. It meant to be used for UI dev on localhost, not for running tests on local host.